pkg/asset/agent/image/oc.go

```diff
@@ -72,19 +75,28 @@ func (r *release) GetBaseIso(log logrus.FieldLogger, releaseImage string, pullSe
 
 	filename := fmt.Sprintf(coreOsFileName, architecture)
 	// Check if file is already cached
-	filePath, err := GetFileFromCache(path.Base(filename), cacheDir)
+	cachedFile, err := GetFileFromCache(path.Base(filename), cacheDir)
 	if err != nil {
 		return "", err
 	}
-	if filePath != "" {
-		// Found cached file
-		return filePath, nil
+	if cachedFile != "" {
+		log.Info("Verifying cached file")
+		valid, err := r.verifyCacheFile(log, image, cachedFile, pullSecret, architecture, mirrorConfig)
+		if err != nil {
+			return "", err
+		}
+		if valid {
+			log.Infof("Using cached Base ISO %s", cachedFile)
+			return cachedFile, nil
+		}
 	}
 
+	// Get the base ISO from the payload
 	path, err := r.extractFileFromImage(log, image, filename, cacheDir, pullSecret, mirrorConfig)
 	if err != nil {
 		return "", err
 	}
+	log.Infof("Base ISO obtained from release and cached at %s", path)
```

Comment:
What's the thinking behind printing the cache path in the info log (vs. debug before)?

Reply:
Its useful to see where the cache file is located in the non-debug case - especially when this file has had to be removed to get past this bug, so I think we should leave it here as Info.
As to why the entire path isn't in debug - we do have the file file there as part of the --path argument, it seemed like the entire path wasn't necessary if it was already being printed in the Info. I'm willing to change if there's a more useful way to do it.